ci/unit_tests/unit_utils.py

```diff
@@ -0,0 +1,42 @@
+
+import os, pytest, unittest
+
+class TestCaseCapture(unittest.TestCase):
```

Comment:
Please name it `BaseTestCaseCapture` or `TestCaseCaptureBase` to clearly state that the purpose of this class is to inherit from it. Consider adding docstring to this class.

Reply:
Fixed